fastlane/lib/fastlane/actions/deploygate.rb

```diff
@@ -17,27 +13,44 @@ def self.is_supported?(platform)
         platform == :ios
       end
 
-      def self.run(options)
-        require 'shenzhen'
-        require 'shenzhen/plugins/deploygate'
+      def self.upload_build(api_token, user_name, ipa, options)
+        require 'faraday'
```

Comment:
`faraday` is a transitive dependency that is pulled in through `spaceship`, maybe we should add it in as an explicit dependency in fastlane's own gemspec.